Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.11.3-stretch
```

Comment:
This needs to be 1.11.4 at minimum (ideally newer)

I'd rather put these in a hack/ subdirectory than clutter the root directory further if we can avoid it, I don't expect most users will use this.
(kind / kubernetes developers will likely have go installed, end users will either `go get` install for convenience, or get a pre-built release binary).